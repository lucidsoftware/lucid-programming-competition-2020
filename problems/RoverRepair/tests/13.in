127
11
16
6
2
10
14
23
1
3
9
empty
13
15
19
30
0
empty
empty
4
7
empty
empty
empty
12
empty
empty
empty
17
20
26
empty
empty
empty
empty
empty
empty
empty
empty
5
empty
8
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
18
empty
22
24
28
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
empty
21
empty
empty
25
27
29
empty
empty
empty
empty
