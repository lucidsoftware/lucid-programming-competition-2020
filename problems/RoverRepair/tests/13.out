Swap nodes 6 and 16
